### PR TITLE
Generate bootstrap and application properties from per module template

### PR DESCRIPTION
**Pull request by sophiaso**

### CODE REVIEW

File: src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java [hunk 1]
```diff
@@ -482,6 +486,12 @@ private void resolveRequestMicroServicesModel(@NonNull ProjectRequest request, @
             serviceModel.put("name", s.getName());
             serviceModel.put("modules", s.getModules());
 
+            if(ServiceNames.isAzureService(service.getName())) {
```
Comment (Incarnation-p-lee) [inline src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java]: space after if

Comment (sophiaso) [reply src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java]: Updated.

File: src/main/java/com/microsoft/azure/springcloudplayground/metadata/ServicePropsResolver.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        }
+    }
+
+    private static String combineAndRemoveDuplicate(List<String> resourceFiles) throws IOException{
+        ResourceLoader resourceLoader = new DefaultResourceLoader();
+        List<String> propLines = new ArrayList<>();
+        for(String propFile : resourceFiles) {
+            List<String> lines = IOUtils.readLines(resourceLoader.getResource(propFile).getInputStream(), Charset.forName("UTF-8"));
+            lines.forEach(line -> {
+                if( line.isEmpty() || !propLines.contains(line)) {
```
Comment (Incarnation-p-lee) [inline src/main/java/com/microsoft/azure/springcloudplayground/metadata/ServicePropsResolver.java]: also space

File: src/main/java/com/microsoft/azure/springcloudplayground/metadata/ServicePropsResolver.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            return combineAndRemoveDuplicate(applicationPropsFiles);
+        } catch (IOException e) {
+            throw new IllegalStateException("Failed to generate application properties file.", e);
+        }
+    }
+
+    private static String combineAndRemoveDuplicate(List<String> resourceFiles) throws IOException{
+        ResourceLoader resourceLoader = new DefaultResourceLoader();
+        List<String> propLines = new ArrayList<>();
+        for(String propFile : resourceFiles) {
```
Comment (Incarnation-p-lee) [inline src/main/java/com/microsoft/azure/springcloudplayground/metadata/ServicePropsResolver.java]: space after `for`

Comment (sophiaso) [reply src/main/java/com/microsoft/azure/springcloudplayground/metadata/ServicePropsResolver.java]: updated.

File: src/main/java/com/microsoft/azure/springcloudplayground/metadata/ServicePropsResolver.java (hunk 2 above)
Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/azure/springcloudplayground/metadata/ServicePropsResolver.java]: I am not sure if we can use `HashSet<String>` for proplines, if so, these can be simplify to `lines.stream().filter(String::isEmpty).forEach(propLines::add)`

**Comment by sophiaso** [comment]
refs #45

**Pull request merged**